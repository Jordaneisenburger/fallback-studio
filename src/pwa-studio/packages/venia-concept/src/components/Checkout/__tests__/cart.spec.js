import React from 'react';
import testRenderer from 'react-test-renderer';
import Cart from '../cart';

jest.mock('../../../classify');

test('renders a Cart component', () => {
    const props = {
        beginCheckout: jest.fn(),
        ready: true,
        submitting: false
    };
    const component = testRenderer.create(<Cart {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
});
