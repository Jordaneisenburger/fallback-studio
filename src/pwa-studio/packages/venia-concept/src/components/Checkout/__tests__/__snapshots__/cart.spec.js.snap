// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Cart component 1`] = `
<div
  className="root"
>
  <button
    className="root_highPriority"
    disabled={false}
    onClick={[MockFunction]}
    type="button"
  >
    <span
      className="content"
    >
      <span
        className="root"
      >
        <svg
          fill="none"
          height={16}
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="2"
          viewBox="0 0 24 24"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            height="11"
            rx="2"
            ry="2"
            width="18"
            x="3"
            y="11"
          />
          <path
            d="M7 11V7a5 5 0 0 1 10 0v4"
          />
        </svg>
      </span>
      <span>
        Checkout
      </span>
    </span>
  </button>
</div>
`;
